extraParams.hxml

```diff
@@ -0,0 +1 @@
+--macro addGlobalMetadata("", "@:build(react.ReactMacro.buildJsxStatic())")
```

Comment:
This makes me sad.

And though it works I'm not sure this is the best approach: we'll have the same problem in other cases where we just use a functional component without using `:jsxStatic`:
```
function render() {
    return jsx('<FunctionalComp />');
}
static function FunctionalComp(props) {
    ...
}
```

Reply:
I agree.
Maybe this is one of several things that could be added in jsx macro, which might need a (partial?) rewrite to handle more complex features.

Reply:
I have a working prototype without extraParams, handled within jsx macro.
I'll clean things up a bit, factorize a little and try to find edge cases (local variables?).

The output will be a little different (but better imo), and still at the end of the generated code:
```javascript
components_Footer.render.displayName = "Footer";
components_Todo.render.displayName = "Todo";
```